public/stylesheet.css

```diff
@@ -1,11 +1,14 @@
 * {
   box-sizing: border-box;
+  font: inherit;
   margin: 0;
+  outline: 0;
   padding: 0;
+  vertical-align: baseline;
 }
 
 :root {
-  font-size: 100%;
+  font-size: 100%; 
```

Comment:
Did you try moving this to the `*` ruleset, similar to Meyer's reset?

Reply:
I would do that, I chose not to modify it as wasn't affecting any styles